main.go

```diff
@@ -26,6 +26,14 @@ var (
 )
 
 func main() {
+	if len(os.Args) > 1 && os.Args[1] == "healthcheck" {
+		if lockFilePresent() {
+			os.Exit(0)
+		}
+
+		os.Exit(-1)
```

Comment:
Use a number over 0 please

Reply:
There is already another usage of `os.Exit(-1)`, that's why I used this.

Reply:
Yes, that was probably my fault, feel free to change that whilst you are in the code.